cylc/rose/utilities.py

```diff
@@ -87,7 +87,23 @@ def get_rose_vars_from_config_node(config, config_node, environ):
     for section in ['env', templating]:
         # Add standard ROSE_VARIABLES
         config_node[section].set(['ROSE_SITE'], rose_site)
+        if 'ROSE_VERSION' in config_node[section].value:
+            user_rose_version = config_node[section].value['ROSE_VERSION']
+            LOG.warning(
+                f'[{section}]ROSE_VERSION={user_rose_version.value} '
+                'from rose-suite.conf will be ignored: '
+                f'ROSE_VERSION set: {ROSE_VERSION}'
```

Comment:
I think this is a bit easier to understand:

```suggestion
                f'Using Rose: {ROSE_VERSION}'
```